agents/s3.js

```diff
@@ -87,7 +87,20 @@ function getSignedFormFields(additionalConditions) {
   };
 }
 
+function listObjects(prefix) {
+  const s3Client = new S3();
+  const params = {
+    Bucket: config.s3.bucket,
+    Delimiter: '/',
+    Prefix: prefix,
```

Comment:
Ah, nice catch, this way we can reuse this method even more, by changing the prefix. Silly me :-/